CI/Dockerfile

```diff
@@ -169,47 +155,75 @@ ENV moab_install_dir=${install_dir}/moab
 
 RUN mkdir -p ${moab_build_dir}/build && \
     cd ${moab_build_dir} && \
-    git clone -b ${MOAB_BRANCH} --depth 1 https://bitbucket.org/fathomteam/moab && \
+    git clone -b ${MOAB_VERSION} --depth 1 https://bitbucket.org/fathomteam/moab && \
     cd build && \
-    cmake ../moab -DCMAKE_INSTALL_RPATH=${hdf5_install_dir}/lib:${moab_install_dir}/lib \
-                  -DCMAKE_BUILD_TYPE=Release \
-                  -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
-                  -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
-                  -DCMAKE_CXX_COMPILER=${CXX} \
-                  -DCMAKE_C_COMPILER=${CC} \
-                  -DENABLE_BLASLAPACK=OFF \
-                  -DBUILD_SHARED_LIBS=ON \
-                  -DENABLE_FORTRAN=OFF \
-                  -DENABLE_PYMOAB=ON && \
-    make -j${ci_jobs} && \
+    cmake ../moab \
+        -DCMAKE_INSTALL_RPATH=${hdf5_install_dir}/lib:${moab_install_dir}/lib \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
+        -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
+        -DCMAKE_CXX_COMPILER=${CXX} \
+        -DCMAKE_C_COMPILER=${CC} \
+        -DENABLE_BLASLAPACK=OFF \
+        -DBUILD_SHARED_LIBS=ON \
+        -DENABLE_FORTRAN=OFF \
+        -DENABLE_PYMOAB=ON && \
+    make -j${CI_JOBS} && \
     make install && \
     cd && \
     rm -rf ${moab_build_dir}
 
+ENV embree_install_dir=${install_dir}/embree
+ENV embree_build_dir=${build_dir}/embree
+
+# Clone and install Embree
+RUN if [ "${DOUBLE_DOWN_VERSION}" != "off" ]; then \
+        mkdir -p ${embree_build_dir}/build && \
+        cd ${embree_build_dir} && \
+        git clone -b ${EMBREE_VERSION} https://github.com/embree/embree && \
+        cd build && \
+        cmake ../embree \
+            -DCMAKE_INSTALL_PREFIX=${embree_install_dir} \
+            -DEMBREE_TASKING_SYSTEM=INTERNAL \
+            -DEMBREE_ISPC_SUPPORT=OFF \
+            -DEMBREE_TUTORIALS=OFF \
+            -DEMBREE_TBB_ROOT=/usr && \
+        make -j${CI_JOBS} && \
+        make -j${CI_JOBS} install && \
+        cd && \
+        rm -rf ${embree_build_dir}; \
+    fi
+
```

Comment:
Embree should be built/installed as part of `external_deps` since it is truly external and we expect it to change less often.

Reply:
I know we discussed this before and my instinct is once again for it to come earlier.  We do change MOAB more frequently than we change Embree, so it makes sense for Embree to come first - and it is "external" as that stage suggests.